container/docker/factory.go

```diff
@@ -49,6 +49,8 @@ var dockerCgroupRegexp = regexp.MustCompile(`.+-([a-z0-9]{64})\.scope$`)
 
 var noSystemd = flag.Bool("nosystemd", false, "Explicitly disable systemd support for Docker containers")
 
+var dockerMetadataEnvs = flag.String("docker_metadata_env", "", "Comma seperated list with names of env variables, which will be exported as metadata (default: empty)")
```

Comment:
Thanks for explaining @dqminh. The flag name doesn't seem to convey the fact that it is a *filter*. Can you fix the naming and description to make it clear that it is a filter?